acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
This doesn't allow the cockroach client to "just work" with boot2docker: it makes an empty temp dir on the client, copies it to the boot2docker vm, then mounts that copy in the container. All the containers can see the same certs directory, but the client (running outside of any container) can't see the certs. We have to generate the certs outside the container and then pass them in; we can't easily get anything created by the containers back out.